logzio/resource_metrics_account.go

```
@@ -0,0 +1,214 @@
+	"github.com/logzio/logzio_terraform_client/metrics_accounts"
+	"github.com/logzio/logzio_terraform_provider/logzio/utils"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+const (
+	metricsAccountId                 string = "account_id"
+	metricsAccountEmail              string = "email"
+	metricsAccountName               string = "account_name"
+	metricsAccountToken              string = "account_token"
+	metricsAccountPlanUts            string = "plan_uts"
+	metricsAccountAuthorizedAccounts string = "authorized_accounts"
+
+	metricsAccountRetryAttempts = 8
+)
+
+// The endpoint resource schema, what terraform uses to parse and read the template
+func resourceMetricsAccount() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMetricsAccountCreate,
+		ReadContext:   resourceMetricsAccountRead,
+		UpdateContext: resourceMetricsAccountUpdate,
+		DeleteContext: resourceMetricsAccountDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			metricsAccountId: {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			metricsAccountToken: {
+				Type:      schema.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+			metricsAccountEmail: {
+				Type:      schema.TypeString,
+				Required:  true,
+				Sensitive: true,
+			},
+			metricsAccountName: {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			metricsAccountPlanUts: {
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			metricsAccountAuthorizedAccounts: {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeInt,
+				},
+				Optional: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func MetricsAccountClient(m interface{}) *metrics_accounts.MetricsAccountClient {
+	var client *metrics_accounts.MetricsAccountClient
+	client, _ = metrics_accounts.New(m.(Config).apiToken, m.(Config).baseUrl)
```

Why this error is ignored?